src/cityreader/cityreader.py

```diff
@@ -20,14 +29,19 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
-    return cities
+  with open('cities.csv', 'r') as csv_file:
+    csv_reader = csv.reader(csv_file)
+    next(csv_reader)
+    for line in csv_reader:
+      cities.append(City(line[0], float(line[3]), float(line[4])))
+  return cities
 
 cityreader(cities)
 
 # Print the list of cities (name, lat, lon), 1 record per line.
+
 for c in cities:
-    print(c)
```

Comment:
if your repr in your city class included the lat and lon, then here you would just need to print c.